demisto_sdk/commands/common/tests/playbook_test.py

```diff
@@ -580,6 +580,12 @@ class TestPlaybookValidator:
         (INVALID_PLAYBOOK_INPUTS_USE, False, True),
     ]
 
+    def test_valid_schema(self, playbook):
```

Comment:
Not sure I understand what this checks that was not checked so far?

Reply:
Yes there's a bit of a mess... I searched for `is_valid_scheme` in the repo and here's what we have:
* In `structure_test.py`:
   * For playbooks there's only `test_scheme_validation_playbook` which does not use TestSuite
   * There are 9 schema tests (with TestSuite) for incident/indicator fields and reputation
   * There are schema tests (with TestSuite) for all **XSIAM** content types
* In different test files, there are 3 tests for generic fields, and 1 test for layout rule

Except for these there are no tests for schemas at all.

Reply:
Meaning we were not validating schema for playbooks?

Reply:
we validate in `test_scheme_validation_playbook`